pkgs/development/tools/misc/semver-tool/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "semver-tool-${version}";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fsaintjacques";
+    repo = "semver-tool";
+    rev = version;
+    sha256 = "0lpwsa86qb5w6vbnsn892nb3qpxxx9ifxch8pw3ahzx2dqhdgnrr";
+  };
+
+  postPatch = ''
+    patchShebangs .
```

Comment:
You should only patch the tests – this will patch `src/semver` with build-time bash as well.

Reply:
Ah, didn't occur to me that build-time bash could be different from runtime. But of course they can (eg if crosscompiling, right?).